### PR TITLE
removed word added Windows Remote Management

### PR DESCRIPTION
**Pull request by RAJU2529**
as per the user report #5040 , so i removed  **Remote Administration** and added **Windows Remote Management**

### CODE REVIEW

**Comment by PRMerger16** [comment]
@RAJU2529 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: WindowsServerDocs/administration/server-core/server-core-manage.md [hunk 1]
```diff
@@ -99,7 +99,7 @@ Where *rulegroup* is one of the following, depending on which snap-in you want t
 > Additionally, certain snap-ins require further configuration before they can connect through Windows Firewall:
 >
 > - Disk Management. You must first start the Virtual Disk Service (VDS) on the Server Core computer. You must also configure the Disk Management rules appropriately on the computer that is running the MMC snap-in.
-> - IP Security Monitor. You must first enable remote management of this snap-in. To do this, at a command prompt, type **Cscript \windows\system32\scregedit.wsf /im 1**
+> - IP Security Monitor. You must first enable remote management of this snap-in. To do this, at a command prompt, type **cscript C\windows\system32\scregedit.wsf /im 1**
```
Comment (illfated) [inline WindowsServerDocs/administration/server-core/server-core-manage.md:102]:
```suggestion
> - IP Security Monitor. You must first enable remote management of this snap-in. To do this, at a command prompt, type **cscript c:\windows\system32\scregedit.wsf /im 1**
```

**Comment by PRMerger19** [comment]
@RAJU2529 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by RAJU2529** [comment]
@PRMerger19 , also verify   #8660   and  #8685

**Review by illfated — APPROVED** [review]
Looks good to me.

**Comment by illfated** [comment]
@RAJU2529 : Please note that @PRMerger16 and @PRMerger19 are scripted bots used to update newly created PRs and issue tickets with a few default actions like labeling and assigning the document author for final review. They also add a default comment, but they are not supposed to respond to comments from us other users (as far as I am aware).

**Comment by illfated** [comment]
#sign-off

**Comment by PRMerger19** [comment]
@illfated: I'm sorry - only the author of this article, @pronichkin, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

**Review by eross-msft — APPROVED** [review]

**Comment by illfated** [comment]
(ref. #821)

**Pull request merged**